runtime/src/accounts_index.rs

```diff
@@ -1312,14 +1312,13 @@ impl<T: IndexValue> AccountsIndex<T> {
     pub(crate) fn scan<'a, F, I>(&'a self, pubkeys: I, mut callback: F)
     where
         // params:
-        //  exists: false if not in index at all
-        //  index in slot list where best slot was found or None if nothing found by root criteria
         //  pubkey looked up
-        //  refcount of entry in index
-        F: FnMut(bool, &SlotList<T>, &'a Pubkey, RefCount) -> AccountsIndexScanResult,
+        // slots_refs is Option<(slot_list, ref_count)>
+        //  slot_list: index in slot list where best slot was found or None if nothing found by root criteria
```

Comment:
this comment was outdated already.
'slots_refs' is None if 'pubkey' is not in accounts index.
slot_list: comes from accounts index for 'pubkey'

Reply:
Updated the comments accordingly.